parser-typechecker/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -369,6 +372,45 @@ loop = do
         stepManyAt = Unison.Codebase.Editor.HandleInput.stepManyAt inputDescription
         stepManyAtM = Unison.Codebase.Editor.HandleInput.stepManyAtM inputDescription
         updateAtM = Unison.Codebase.Editor.HandleInput.updateAtM inputDescription
+        manageLinks :: Var v0
+                    => [(Path', NameSegment.HQSegment)]
+                    -> (Path', NameSegment.HQSegment)
+                    -> (forall r. Ord r
+                        => (r, Reference, Reference)
+                        ->  Branch.Star r NameSegment
+                        ->  Branch.Star r NameSegment)
+                    -> MaybeT (StateT (LoopState m v0) (F m (Either Event Input) v0)) ()
+        manageLinks srcs mdValue op = do
+          let srcle = toList . getHQ'Terms =<< srcs
+              srclt = toList . getHQ'Types =<< srcs
+              mdValuel = toList (getHQ'Terms mdValue)
+          case (srcle, srclt, mdValuel) of
+            (srcle, srclt, [Referent.Ref mdValue]) -> do
+              mdType <- eval $ LoadTypeOfTerm mdValue
+              case mdType of
+                Nothing -> respond $ LinkFailure input
+                Just ty -> do
+                  let steps =
+                        second (const . step $ Type.toReference ty)
+                        . first (Path.unabsolute .
+                                   Path.toAbsolutePath currentPath') <$> srcs
```

Comment:
The conflicting edit was just to replace `Path.toAbsolutePath currentPath'` with `resolveToAbsolute`.